configure.ac

```
@@ -202,6 +204,151 @@ AS_CASE([$with_gc],
     )
 AC_MSG_RESULT([$with_gc])
 
+dnl
+dnl User setting: disable guards (race protection)
+dnl
+
+AC_ARG_ENABLE([guards],
+    [AS_HELP_STRING([--enable-guards],
+        [disable race protection in HPC-GAP])],
+    [enable_guards=$enableval],
+    [enable_guards=no])
+
+AS_IF([[test "x$enable_guards" == "xyes"]], [
+    AC_DEFINE([USE_HPC_GUARDS], [1], [define as 1 if guards should be checked])
+])
+AC_MSG_CHECKING([whether to use race protection in HPC-GAP])
```

Again: the option is `--enable-guards`; the `AC_DEFINE` says "guards should be checked". So I'd refer to "guards" here, too, so that when reading the message in a configure log, it immediately is clear what it relates to.
```suggestion
AC_MSG_CHECKING([whether to use guards for race protection in HPC-GAP])
```

---

Also fixed.